test/fixtures/bootstrappolicy/bootstrap_cluster_roles.yaml

```diff
@@ -1209,6 +1209,13 @@ items:
     verbs:
     - create
     - delete
+  - apiGroups:
+    - ""
```

Comment:
@deads2k this seems odd, is this really any different than `- apiGroups: null` ?